src/loaders/png/tvgPngLoader.cpp

```diff
@@ -51,6 +51,18 @@ bool PngLoader::open(const string& path)
     return true;
 }
 
+bool PngLoader::open(const char* data, uint32_t size, bool copy)
```

Comment:
Did you verify read() works fine even after data is freed?
I think it needs to copy the data if copy == true.

Reply:
The change was tested. There is no need to copy as task aren't used for PngLoader.

Reply:
@mmaciola  like this sequence?
1. picture->open()
2. free(data)
3. picture->push()

Reply:
The test scenario is below. Works ok
```
    string path(EXAMPLE_DIR"/logo.png");

    //Open
    ifstream file(path);
    if (!file.is_open()) return ;
    auto size = sizeof(uint32_t) * (1000*1000);
    auto data = (char*)malloc(size);
    file.read(data, size);
    file.close();

    //Picture
    auto picture = tvg::Picture::gen();
    picture->load(data, size, false);

    //free and push
    free(data);
    canvas->push(move(picture));
```

Reply:
@mmaciola good thanks!